config/defaults_test.go

```diff
@@ -54,6 +54,7 @@ var _ = Describe("Defaults", func() {
 					RunImage:   "gcr.io/grpc-fake-project/test-infra/java",
 				},
 			},
+			KillAfter: "20",
```

Comment:
Perhaps we want to validate the format of KillAfter. Perhaps it should be of type time.Duration, and converted to seconds for the environment variables. Or perhaps it should always be a number representing the duration in seconds, to keep it simple.

If we choose to use the "number of seconds", perhaps we should call it KillAfterSeconds.

Reply:
If it is # of seconds, let's make it an integer and not a string. I'd prefer time.Duration... but I'm okay with integer seconds. I think string seconds is a tad confusing. The timeout command's documentation says it supports the same suffixes as Go, so (time.Duration).String should give the right format if supplied seconds, minutes, hours or days. https://man7.org/linux/man-pages/man1/timeout.1.html

Reply:
I named it as KillAfterSeconds, so that we can use integer second. I decided to use int64 so that we would need to introduce new package. Is there any difference in terms of using int64 or duration in the respect of validation?

Reply:
I think that, since it is a duration, it is less confusing to actually make it of type Duration. Then we can also just call it "KillAfter" etc, without the seconds. When we create the KILL_AFTER environment variable, at this point I would convert to .Seconds() to avoid issues (a duration of "30m20s" is valid in Go but not as an argument to "timeout").

Reply:
Make scene, I will make it less confusing by using duration instead of integer.